lib/widgets/background.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart'; 
+
+
+class BackGround extends StatelessWidget {
```

Comment:
Background is a single word

```suggestion
class Background extends StatelessWidget {
```